### DISCUSSION TITLE
CVE-2023-46298 Patch For Next 12?

### DISCUSSION THREAD

**Discussion by pihish** [vercel/next.js#59029] [Help]
### Summary

https://www.cve.org/CVERecord?id=CVE-2023-46298 was reported [here](https://github.com/vercel/next.js/issues/45301) and patched [here](https://github.com/vercel/next.js/pull/54732).  The fix was applied to all NextJS releases after **13.4.20-canary.13**.  Are there any plans to apply the same fix to Next 12 or are changes to Next 12 no longer being added?  We tried upgrading our NextJS service to 13 and were seeing some significant issues with memory leaks and performance.

**Comment by walkerdepaula** [comment]
I'm with the same problem in my projects. Update Next to 13 its not so simple. @pihish i found [this article](https://medium.com/@mares.viktor/deep-diving-into-cve-2023-46298-resource-exhaustion-in-next-js-7f4312d0cb3f) in Medium and apparently we can configure the next.config.js to return cache-control: no-cache header for SSR paths.

I think that should fix the problem until we can update to the last version or util Vercel apply the same fix to versions 12 of Next.
